tokio-trace/tokio-trace-core/src/metadata.rs

```diff
@@ -142,6 +161,7 @@ impl<'a> Metadata<'a> {
         line: Option<u32>,
         field_names: &'static [&'static str],
         callsite: &'static Callsite,
+        callsite_kind: &'static CallsiteKind,
```

Comment:
I don't think it's necessary to pass an `&'static CallsiteKind` --- it can probably just be moved into this function:
```suggestion
        callsite_kind: CallsiteKind,
```